src/main/features/dashboard/views/macro/claimStatus/claimPartAdmitted.njk

```diff
@@ -0,0 +1,102 @@
+  <p>
+    {{ t('We’ve emailed {{ claimantName }} telling them when and how you said you paid the claim.',
+      { claimantName: claim.claimData.claimant.name }) }}
+  </p>
+  <p>
+    {{ t('We’ll contact you to let you know how they respond. They can confirm you’ve paid and the claim is ' +
+      'settled, or they can proceed with it.') }}
+  </p>
+  <p>
+    {{ _downloadResponseLink(claim) }}
+  </p>
+{% endmacro %}
+
+{% macro claimPartAdmittedAlreadyPaidForClaimantClaimDetails(claim) %}
+  {% if claim.response.paymentDeclaration.paidDate %}
+    {% set paymentDate = claim.response.paymentDeclaration.paidDate | date %}
+  {% endif %}
+  <h2 class="heading-small">{{ t('The defendant’s response') }}</h2>
+  <p>
+    {{ t('{{ defendantName }} says they paid you {{ amount }} on {{ paymentDate }}.',
+      { defendantName: claim.claimData.defendant.name, amount: claim.response.amount | numeral, paymentDate: paymentDate }) }}
+  </p>
+  <p> {{ t('You can accept or reject this response.') }}</p>
+  <p>
+    {{ t('You need to respond before 4pm on {{ respondDate}} or your claim won’t proceed.', {respondDate: claim.respondToResponseDeadline | date}) }}
+  </p>
+{% endmacro %}
+
+{% macro claimPartAdmittedToOweForDefendantDashboard(claim) %}
+  {{ t('You’ve admitted part of the claim.',
+    { defendantName: claim.claimData.defendant.name }) }}
+{% endmacro %}
+
+{% macro claimPartAdmittedToOweForClaimantDashboard(claim) %}
+  {{ t('The defendant believes they owe you {{ amount }}. You can accept or reject that this is the amount owed.',
+    { amount: claim.response.amount | numeral }) }}
+{% endmacro %}
+
+{% macro claimPartAdmittedToOweForDefendantClaimDetails(claim) %}
+  {% if claim.response.paymentIntention.paymentDate %}
+    {% set paymentDate = claim.response.paymentIntention.paymentDate | date %}
+  {% endif %}
+
+  {% if claim.response.paymentIntention.paymentOption === domain.PaymentOption.IMMEDIATELY %}
+    {% set paymentMethod = 'immediately' %}
+  {% elseif claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE %}
+    {% set paymentMethod = 'on ' + paymentDate %}
+  {% elseif  claim.response.paymentIntention.paymentOption === domain.PaymentOption.INSTALMENTS %}
+    {% set repaymentPlan = claim.response.paymentIntention.repaymentPlan %}
+    {% set amount = repaymentPlan.instalmentAmount | numeral %}
+
+    {% if repaymentPlan.paymentSchedule === domain.PaymentSchedule.EVERY_MONTH %}
+      {% set paymentSchedule = 'every month' %}
+    {% elif repaymentPlan.paymentSchedule === domain.PaymentSchedule.EVERY_TWO_WEEKS %}
+      {% set paymentSchedule = 'every two weeks' %}
+    {% elif repaymentPlan.paymentSchedule === domain.PaymentSchedule.EVERY_WEEK %}
+      {% set paymentSchedule = 'every week' %}
+    {% endif %}
+
+    {% set paymentMethod =  amount + ' ' + paymentSchedule + ' starting '+ repaymentPlan.firstPaymentDate |date %}
```

Comment:
Seems like another place where I can see this. Please extract so that we have one place to make change, no duplicates please.